apps/src/templates/projects/publishDialog/PublishDialog.jsx

```diff
@@ -2,11 +2,13 @@ import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
 import Radium from 'radium'; // eslint-disable-line no-restricted-imports
-import BaseDialog from '../../BaseDialog';
-import DialogFooter from '../../teacherDashboard/DialogFooter';
-import Button from '../../Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
Curious about moving from relative to absolute paths here. I see both used frequently. I feel like this change is easier to read since the former version has us going out multiple levels. Are there any other conventions to be aware of here?

Reply:
I think we lean towards the `@cdo/...` version in general (I agree it is more readable). Since I was adding new imports I decided to update all of them in this file.